src/Kernel.php

```diff
@@ -17,16 +17,19 @@
 use Sulu\Component\HttpKernel\SuluKernel;
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\HttpCache\HttpCache;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
 
 class Kernel extends SuluKernel implements HttpCacheProvider
 {
-    use HttpCacheAware;
```

Comment:
Should also be removed from `use` statements, right? And just curious: Why did we change that?

Reply:
Ah, have seen the comment from @alexander-schranz...